TampermonkeyScriptForMidJourney.js

```diff
@@ -91,25 +100,53 @@
 
                        setTimeout(() => {
                            $("button[title='Save with prompt']").click();
-
                            $("button[title='Close']").click();
                        }, 800);
+
+                       setTimeout(() => {
+                           prompt = window.overElement.parents("div[role='gridcell']").find("p._promptText_").text();
+                           const filename = console.logs.pop()[1]
+                           console.logs.length = 0
+                           savePrompt(filename, prompt);
+                       }, 100);
```

Comment:
Wouldn't this delay cause it to save the wrong text file? Specifically the text file from the image before, since the actual "save" button click is only happening in 800ms and yours triggers in 100ms... or am I reading this wrong?

Reply:
Maybe it would be better to put your setTimeout inside of the image save setTimeout? since the next save isn't going to happen until 800ms later anyway... it might also be ok to set the delay to 200ms just to make sure it works on slower connections/PCs too